### PR TITLE
Update access-admin-portal.md

### PR DESCRIPTION
**Pull request by mounicabattula**
With the SEP portal migration to MEM/MMD portal, Support Partners can use MMD/MEM portal to create elevations and escalations. Adding the MMD Support Partner role in the list to MMD built-in roles list.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/get-started/access-admin-portal.md [hunk 1]
```diff
@@ -46,9 +46,10 @@ If you need help with assigning Azure Active Directory roles, see [Azure AD buil
...
-|Microsoft Managed Desktop Service Reader | When assigned to a user, this role gives the admin **read-only permissions to features not related to security** in the Microsoft Managed Desktop Admin portal. |
+|Microsoft Managed Desktop Service Administrator  | When assigned to a user, this role gives the admin **read and write permissions to Microsoft Managed Desktop features not related to security** in the Microsoft Managed Desktop Admin portal.  |
+|Microsoft Managed Desktop Service Reader | When assigned to a user, this role gives the admin **read-only permissions to Microsoft Managed Desktop features not related to security** in the Microsoft Managed Desktop Admin portal. |
 |Microsoft Managed Desktop Security Manager |When assigned to a user, this role gives that admin **read and write permissions only for security related features** in the Microsoft Managed Desktop Admin portal.   |
+|Microsoft Managed Desktop Support Partner |When assigned to a user, this role gives that admin **read and write permissions only for creating and managing elevation requests, and support partner engaged escalation requests** in the Microsoft Managed Desktop Admin portal.   |
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/access-admin-portal.md]:
```suggestion
|Microsoft Managed Desktop Support Partner |When assigned to a user, this role gives the admin **read and write permissions only for creating and managing elevation requests and support partner engaged escalation requests** in the Microsoft Managed Desktop Admin portal.   |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@mounicabattula Please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jaimeo Please merge this PR. Thanks

**Comment by jaimeo** [comment]
I'm not in this area anymore (article authorship has not been updated yet). Contact @dougeby

**Comment by yogkumgit** [comment]
@dougeby Please merge this PR. Thanks

**Pull request merged**